hr_timesheet_activity_begin_end/__openerp__.py

```diff
@@ -1,36 +1,22 @@
 # -*- coding: utf-8 -*-
-#
-#
-#    Authors: Guewen Baconnier
-#    Copyright 2015 Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# Copyright 2015 Camptocamp SA (author: Guewen Baconnier)
+# Copyright 2017 Martronic SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {'name': 'Timesheet Activities - Begin/End Hours',
- 'version': '8.0.1.0.0',
- 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'version': '9.0.1.0.0',
+ 'author': 'Camptocamp,Martronic SA,Odoo Community Association (OCA)',
  'license': 'AGPL-3',
  'category': 'Human Resources',
- 'depends': ['hr_timesheet_sheet',
-             ],
+ 'depends': [
+     'hr_timesheet_sheet',
+     'float_time_hms',
```

Comment:
@Martronic-SA where could we find the `float_time_hms` module ?

Reply:
Sorry this was our own version of the module with float_time_hms (means Hour Minute Second).
You can find the module here:
https://github.com/jordillinares/addons
As long as we are using the activity begin end module in telephonic  hotline, we wanted to have something more accurate than minutes.